config/manifests/rhpam/bases/rhpam-kogito-operator.clusterserviceversion.yaml

```diff
@@ -7,7 +7,8 @@ metadata:
     categories: Integration & Delivery
     certified: "true"
     containerImage: registry.stage.redhat.io/rhpam-7/rhpam-kogito-rhel8-operator:0.0.0
-    description: RHPAM Kogito Operator for deployment and management of Kogito services.
+    description: IBM BAMOE Kogito Operator for deployment and management of Kogito
+      services.
     operators.openshift.io/valid-subscription: '["Red Hat Process Automation Manager",
```

Comment:
what will be the subscription ? :S

Reply:
I don't know =/

Reply:
This should be `Business Decision Manager` right?